guide/C/ch_invest.xml

```diff
@@ -1359,76 +1351,66 @@ Income
-              </para>
-            </listitem>
-
-            <listitem>
-              <para>Change to that directory, open a root shell
-              </para>
-            </listitem>
-
-            <listitem>
-              <para>Run the command <command>gnc-fq-update</command>
-              </para>
-            </listitem>
-          </itemizedlist>
-          This will launch a <application>Perl</application> CPAN update session that will go out
-          onto the internet and install the <application>Finance::Quote</application> module on your
-          system. The gnc-fq-update program is interactive, however, with most systems you should be
-          able to answer <quote>no</quote> to the first question: <quote>Are you ready for manual
-          configuration? [yes]</quote> and the update will continue automatically from that point.
-        </para>
-
-        <para>After installation is complete, you should run the <quote>gnc-fq-dump</quote> test program, in the
-          same directory, distributed with &app; to test if
-          <application>Finance::Quote</application> is installed and working properly.
-        </para>
+        <title>Installing &app-fq;</title>
+        <para>The process of installing &app-fq; depends on the system. </para>
+
+        <procedure>
+          <title>Installing &app-fq; on &win;</title>
+          <step><simpara>Close &app;.</simpara></step>
+          <step><simpara>Run <menuchoice>
+                               <guimenu>Start</guimenu>
+                               <guisubmenu>&app;</guisubmenu>
+                               <guimenuitem>Install Online Price Retrieval</guimenuitem></menuchoice>.
+            </simpara>
+          </step>
+        </procedure>
+
+        <procedure>
+          <title>Installing &app-fq; on &mac;</title>
+          <step><simpara>Install Xcode if it is not installed.</simpara>
+              <simpara>XCode is an optional item from your &mac; distribution DVD.</simpara>
+          </step>
+          <step><simpara>Run the <emphasis>Update Finance Quote</emphasis> app in the &app; dmg.</simpara> 
+              <para>You can run it from the dmg or copy it to the same
+                folder to which you copied &app;.
+                It will open a Terminal window and run a script for you
+                which will ask lots of questions. Accept the default for each unless you know what
+                you&rsquo;re doing.
+            </para>
+          </step>
+        </procedure>
+
+        <procedure>
+          <title>Installing &app-fq; on &lin;</title>
+          <step><simpara>Close any running &app; instances.</simpara></step>
+          <step><simpara>Locate the folder where &app; is installed by searching for <filename>gnc-fq-update</filename>.</simpara></step>
+          <step><simpara>Change to that directory, open a root shell.</simpara></step>
```

Comment:
By default, it is installed in make's PREFIX/bin, which is usually in environment's $PATH. Then you need only to enter <command>sudo gnc--fq-update</command>. Only if it is not found because you had installed it to somewhere else, the two steps are required.

Reply:
35c8e580